### PR TITLE
767843484 - AADSUser on the design system, when I click the example form story, I see an input with additional information or an error message below

### PR DESCRIPTION
**Pull request by elio-stalteri**

### CODE REVIEW

File: src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    error: "error",
+    caption: "caption",
+    isError: false,
+    isFocus: false,
+  }
+  const component = mount(<HelperErrorMessage {...propsMock} />)
+  const StyledErrorMessageComponent = mount(<StyledErrorMessage>test</StyledErrorMessage>)
+  const StyledHelperCaptionComponent = mount(<StyledHelperCaption>test</StyledHelperCaption>)
+
+  it("HelperErrorMessage sould match last snapshot", () => {
```
Comment (alexwong-crc) [inline src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js:18]: the test mentions snapshot but I don't see a snapshot test. Also, this test is not clear at all. I have no idea what your are testing and why you are changing the props all the time. Can you split these into smaller tests with easier to read test names.

File: src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+describe("HelperErrorMessage Component Test", () => {
+  const propsMock = {
+    error: "error",
+    caption: "caption",
+    isError: false,
+    isFocus: false,
+  }
+  const component = mount(<HelperErrorMessage {...propsMock} />)
+  const StyledErrorMessageComponent = mount(<StyledErrorMessage>test</StyledErrorMessage>)
+  const StyledHelperCaptionComponent = mount(<StyledHelperCaption>test</StyledHelperCaption>)
```
Comment (alexwong-crc) [inline src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js:16]: Why are the`StyledErrorMessageComponent` and the `StyledHelperCaptionComponent ` in the global scope, when it is only used in the tests below. Can you move them to the relevant it statements.

File: src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+describe("HelperErrorMessage Component Test", () => {
+  const propsMock = {
+    error: "error",
+    caption: "caption",
+    isError: false,
+    isFocus: false,
+  }
+  const component = mount(<HelperErrorMessage {...propsMock} />)
+
+  it("HelperErrorMessage should should show a caption message when isFocus is true", () => {
```
Comment (alexwong-crc) [inline src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js:16]: can you add a test for when isFocus is true and isError is true, but it should only show the error not the caption.

Comment (elio-stalteri) [reply src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js:19]: done

File: src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    ).toBeTruthy()
+  })
+
+  it("HelperErrorMessage should show an error message when isError is true and isFocus is true", () => {
+    expect(
+      component.contains(<StyledErrorMessage className="ErrorMessage">error</StyledErrorMessage>),
+    ).toBeFalsy()
+    component.setProps({ ...propsMock, isError: true, isFocus: true })
+    expect(
+      component.contains(<StyledErrorMessage className="ErrorMessage">error</StyledErrorMessage>),
```
Comment (alexwong-crc) [inline src/components/organism/Formik/HelperErrorMessage/HelperErrorMessage.test.js:48]: can you assert that caption is not showing.

**Review by alexwong-crc — APPROVED** [review]

**Pull request merged**